ropsten-beacon-chain/README.md

```diff
@@ -0,0 +1,53 @@
+# Ropsten beacon chain
+TTD: `43531756765713534`
+
+GENESIS_FORK_VERSION: `0x80000069`
+
+ENR for consensus clients:
+```yaml
+  - "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk"
+```
+
+
+Altair will fork at epoch 500 and merge epoch at 750.
+```yaml
+genesis_time: 1653923100
+genesis_state_root: 0x3b807197e5e874635dd3947091f1fa6bdd1271080a1a818b03e9b906bd593d39
+genesis_latest_block_header:
+  slot: 0
+  proposer_index: 0
+  parent_root: 0x0000000000000000000000000000000000000000000000000000000000000000
+  state_root: 0x0000000000000000000000000000000000000000000000000000000000000000
+  body_root: 0xccb62460692be0ec813b56be97f68a82cf57abc102e27bf49ebf4190ff22eedd
+genesis_block_root_no_state_root: 0xeade62f0457b2fdf48e7d3fc4b60736688286be7c7a3ac4c9a16a5e0600bd9e4
+genesis_block_root_updated_state_root: 0x93fb1b676c70ba75f52cd899cdb842a0c3e1f9848dee697696874dfd77ac4366
+genesis_validators_root: 0x44f1e56283ca88b35c789f7f449e52339bc1fefe3a45913a43a6d16edcd33cf1
+genesis_validators_count: 100012
+genesis_active_validators_count: 100012
```

Comment:
`MIN_GENESIS_ACTIVE_VALIDATOR_COUNT` is 10000 in `ropsten-beacon-chain/config.yaml`

Reply:
Nvm, that should be fine. We can deposit > `MIN_GENESIS_ACTIVE_VALIDATOR_COUNT`